.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,9 @@
-https://github.com/nimblehq/flutter_templates/issues/??
+Note: for a release PR, append this parameter `?template=release_template.md` to the current URL to apply the release PR
+template, e.g. `{Github PR URL}?template=release_template.md`
+
+--
+
+- Solves #
```

Comment:
FYI: We should use `Solves #` to automatically link & close the ticket after merging.